deployment/charts/values.yaml

```diff
@@ -0,0 +1,20 @@
+# Default values for fuel core for environment
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+app:
+  name: fuel-core
+  replicas: 1
+  port: 80
+  targetport: 4000
+  image:
+    repository: ghcr.io/fuellabs/fuel-core
+    tag: latest
```

Comment:
is there a way to make this configurable based on an env var or something similar? We will want to be able to control which images are selected.

Reply:
@Voxelot - the values.yaml can be updated by the user  to populate the helm templates however they wish (https://helm.sh/docs/chart_template_guide/values_files/)  - so the line 12 can be updated and then the helm chart redeployed via (https://github.com/FuelLabs/fuel-core/blob/roy-fuel-eks-helm-charts/deployment/scripts/fuel-core-deploy.sh)

That being said I will work on updating that apart of the fuel-deployment pipeline so helm chart in this issue- (https://github.com/FuelLabs/fuel-deployment/issues/3 .